Xamarin.Forms.Platform.iOS/EventTracker.cs

```diff
@@ -292,6 +308,15 @@ UIPinchGestureRecognizer CreatePinchRecognizer(Action<UIPinchGestureRecognizer>
 			return result;
 		}
 
+		UISwipeGestureRecognizer CreateSwipeRecognizer(SwipeDirection direction, Action<SwipeDirection> action, int numFingers = 1)
```

Comment:
I think we're going to need to enable `ShouldRecognizeSimultaneously` here so we don't have the same issue we have in #2296.

Reply:
Good shout, Cheers!